PopPUNK/trees.py

```diff
@@ -176,46 +190,49 @@ def mst_to_phylogeny(mst_network, names):
     #
     # MST graph -> phylogeny
     #
+    
+    if use_gpu and not gpu_lib:
+        sys.stderr.write('Unable to load GPU libraries; exiting\n')
+        sys.exit(1)
+    
     # Define sequences names for tree
     taxon_namespace = dendropy.TaxonNamespace(names)
     # Initialise tree and create nodes
     tree = dendropy.Tree(taxon_namespace=taxon_namespace)
-    tree_nodes = {v:dendropy.Node(taxon=taxon_namespace[int(v)]) for v in mst_network.get_vertices()}
+    if use_gpu:
```

Comment:
Combine this if block with the one immediately below

Reply:
Done in f9fa8ef.